osu.ElasticIndexer/HighScoreIndexer.cs

```diff
@@ -19,8 +19,6 @@ public class HighScoreIndexer<T> : IIndexer where T : Model
         public long? ResumeFrom { get; set; }
         public string Suffix { get; set; }
 
-        private readonly ElasticClient elasticClient = new ElasticClient(new ConnectionSettings(new Uri(AppSettings.ElasticsearchHost)));
```

Comment:
might be a bit cleaner to leave this variable and assign the static here

```
private readonly ElasticClient elasticClient = AppSettings.ELASTIC_CLIENT; // use static instance to avoid socket leakage.
```